from_cpython/Python/mystrtoul.c

```diff
@@ -92,7 +92,7 @@ static int digitlimit[] = {
 **              exceptions - we don't check for them.
 */
 unsigned long
-PyOS_strtoul(register char *str, char **ptr, int base)
+PyOS_strtoul(register const char *str, char **ptr, int base)
```

Comment:
While the signature looks strange with the mixed constness I tried to match the signature of ```strtoul``` http://www.cplusplus.com/reference/cstdlib/strtoul/
I will revert this change and cast away the constness at the caller side.